src/site/stages/build/options.js

```diff
@@ -31,6 +31,8 @@ const COMMAND_LINE_OPTIONS_DEFINITIONS = [
   { name: 'asset-source', type: String, defaultValue: assetSources.LOCAL },
   { name: 'content-directory', type: String, defaultValue: defaultContentDir },
   { name: 'pull-drupal', type: Boolean, defaultValue: false },
+  { name: 'use-cms-export', type: Boolean, defaultValue: false },
+  { name: 'cms-export-dir', type: String, defaultValue: false },
```

Comment:
**Food for thought for later:**
:thinking: At some point, we may want to add a `description` to these. It doesn't do anything in the `command-line-args` library, but it might be helpful for anybody reading these later. Comments work too. :man_shrugging: